### DISCUSSION TITLE
Generate requests based on existing OpenAPI spec

### DISCUSSION THREAD

**Discussion by chbndrhnns** [jonaslu/ain#7] [Ideas]
This is what the UI-based tools can do: they read the spec file and create requests for you, either stub requests if no examples are specified or they even take the examples from the spec file as payloads.

I would like to understand if this would be of interest for this tool, as well.

**Comment by jonaslu** [comment]
Hi, yes it would be of interest. I'd  rather make that an own binary (perhaps named aingen?) than put it into ain. This binary would know to write out template.ain files from many formats . Ideally it will be bundled with ain so the come in the same install.

I haven't looked into how much work it would be, but I'll probably do that in the not-too-distant future. Or I'd be open to help out if someone else did too...
